### PR TITLE
Solved - Issue #741 Upload model form on view page needs adjustment o…

### PR DESCRIPTION
**Pull request by HRIDYANSHU054**
Solved Issue: Upload model form on view page needs adjustment on mobile view

## Summary
More Details. Added media queries to handle this in components\modalbox\modalbox.css 

## Motivation
Previous version of the modal form was not adapting well to mobile screens discouraging the application usage on mobile devices.

## Testing
The responsiveness testing was done using chrome developer tools
Following are the results
tabletUIDistorted
Tablet UI Distortion

tabletUIsolved
Tablet UI solved

mobileUIDistorted
Mobile UI Distortion

mobileUIsolved
Mobile UI solved

desktopUI
Desktop UI

### CODE REVIEW

File: components/modalbox/modalbox.css [hunk 1]
```diff
@@ -22,11 +22,27 @@
   padding: 0;
   border: 1px solid #888;
   width: 80%;
+  
+  @media only screen and (max-width: 1100px) {
+        width: 50%;
+  }
+
+  @media only screen and (max-width: 600px) {
+        width: 85%;
```
Comment (birm) [inline components/modalbox/modalbox.css:31]:
```suggestion
        width: 85% !important;
```

File: components/modalbox/modalbox.css [hunk 2]
```diff
@@ -22,11 +22,27 @@
   padding: 0;
   border: 1px solid #888;
   width: 80%;
+  
+  @media only screen and (max-width: 1100px) {
+        width: 50%;
```
Comment (birm) [inline components/modalbox/modalbox.css:27]:
```suggestion
        width: 50% !important;
```

**Review by birm — CHANGES_REQUESTED** [review]
This seems like a good idea, but I can't get it to actually render as expected without these "important" rules

**Comment by HRIDYANSHU054** [comment]
@birm I appreciate that you brought this to my notice this was blunder on my end. Please check now.

**Review by birm — APPROVED** [review]
That did it! Thank you!

**Pull request merged**